### ISSUE TITLE
Broadcasting to all connected clients

### ISSUE THREAD

**Issue by imrankhan17** [miguelgrinberg/flask-sock#9]
Hi, is there a way to get a list of all connected clients and then to broadcast a message to them?  This is the same functionality as [this](https://websockets.readthedocs.io/en/stable/intro.html?highlight=broadcast#registration).

**Comment by miguelgrinberg** [comment]
No. This package just offers the basic connectivity support. Higher level packages that use this as a dependency do offer broadcasting as an option (such as Socket.IO or Turbo-Flask).

**Comment by PizzaRules668** [comment]
I mean you can make a list and them add and remove them from the list. It might not be idea but it works. Then loop over the list to send data

**Comment by miguelgrinberg** [comment]
Correct. That is what those other packages I mentioned do.
